src/App.css

```diff
@@ -14,19 +14,57 @@ html, body, #root, #app {
 }
 
 .toolbar-container {
-  text-align: center;
   position: fixed;
+  align-items: right;
+  text-align: right;
   top: 10px;
-  right: 10px;
+  right: 5%;
   z-index: 2;
   user-select:none;
   -webkit-tap-highlight-color:rgba(0,0,0,0);
 }
 
+.hidden {
+  display: none;
+}
+
+#leftArrow {
```

Comment:
Please use the lowercase-with-dash convention that is used for CSS ids and classes in this project.

Reply:
Got it.